moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/current_state_monitor.h

```diff
@@ -179,8 +179,19 @@ class CurrentStateMonitor
     return error_;
   }
 
-  /** @brief Allow the joint_state arrrays velocity and effort to be copied into the robot state
-   *  this is useful in some but not all applications
+  /** @brief Check whether the robot is stopped. Two successive states are checked for position change,
+   *  and velocities are checked.
```

Comment:
What if the trajectory contains a phase where the robot doesn't physically move, but is still executing the trajectory?

This is very common in certain applications in which the process the robot is performing takes a bit longer.

Reply:
that's a good point, I'm not sure how I could check that easily. Maybe RobotState should have a flag for whether it's currently executing a trajectory

Reply:
Or if the robot is somewhat slow in starting. There were some funny bugs in industrial_robot_client due to using heuristics like this. I'm not saying this is necessarily a bad idea just thinking you might also want to consider information from `TrajectoryExecutionMonitor` (which should return success only once the movement has actually finished)

Reply:
OK, you've convinced me this won't work for trajectory monitoring. Thanks for the pointer to `TrajectoryExecutionMonitor`.

We're actually using it to monitor a switch from streaming motion ("jogging") to trajectories, which is a little bit different. But I guess this functionality belongs in ros_control, probably.

Reply:
I'll give it a couple more days to see if anybody else has comments, then close the issue